### PR TITLE
Use new features on SDK to simplify the smart contract

### PR DESCRIPTION
**Pull request by emmanuelm41**
:link: [zboto Link](https://app.clickup.com/t/2p0kymt)

### CODE REVIEW

File: assembly/index.ts [hunk 1]
```diff
@@ -39,32 +38,32 @@ function register(rawParams: ParamsRawResult): void {
 // @ts-ignore
 @export_method(3)
 // `transfer` transfer a domain to a new address
-function transfer(rawParams: ParamsRawResult): void {
-  const params = TransferParams.fromRaw(rawParams.raw)
-
+function transfer(name: string, addr: Uint8Array): void {
```
Comment (rllola) [inline assembly/index.ts:41]: Nice!

File: tests/browser/lib/base64.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+export const base64ArrayBuffer = (arrayBuffer:ArrayBuffer) => {
```
Comment (rllola) [inline tests/browser/lib/base64.ts:1]: Can't we use builtin base64 encoder ?

**Review by rllola — APPROVED** [review]

File: tests/browser/lib/base64.ts (hunk 2 above)
Comment (emmanuelm41) [reply tests/browser/lib/base64.ts:1]: What is the builtin encoder?

Comment (rllola) [reply tests/browser/lib/base64.ts:1]: window.btoa I think

**Pull request merged**